ts/bindings/typescript.ts

```diff
@@ -135,17 +135,27 @@ export const typeGenerator = (schema: Schema): string => {
       ({ name, schema }) =>
         `@param ${name} {${schemaToType(schema)}} ${schema.description}`
     ),
```

Comment:
I think this bit isn't quite right any more as it produces a mismatch between the function arity. E.g. `Article`:

``` ts
/**
 * Create a `Article` node
 * @param authors {Array<Person | Organization>} The authors of this creative work.
 * @param title {string | Array<Node>} The title of the creative work.
 * @param other Other properties
 * @returns {Article}
 */
export const article = (
  authors: Array<Person | Organization>,
  props: { title: string | Array<Node> } & OptionalProps<Article>
): Article => ({
  authors,
  ...compact(props),
  type: 'Article'
})
```

Note that the `@param title` is in the Doc Comment, but not part of the function argument.